hasher/Cargo.toml

```diff
@@ -18,4 +18,15 @@ bitvec = "1.0.0"
 serde = { version = "1.0", features = ["derive"] }
 
 [dev-dependencies]
+criterion = {version = "0.3", features = ["html_reports"]}
+lazy_static = "1"
+lockfree-object-pool = "0.1"
 serde_json = { version = "1.0" }
+
+[profile.bench]
+lto = true
```

Comment:
wondering if this would be good to add to the other benchmarks we have :o

Reply:
I can do that, also added `codegen-units= 1`. Plz note that both [lto](https://doc.rust-lang.org/cargo/reference/profiles.html#lto) and [codegen-units](https://doc.rust-lang.org/cargo/reference/profiles.html#codegen-units) slows down the build